internal/auth/auth.go

```diff
@@ -26,7 +26,9 @@ func WithAuthedUser(ctx context.Context, user *DelphisAuthedUser) context.Contex
 
 func GetAuthedUser(ctx context.Context) *DelphisAuthedUser {
 	val := ctx.Value(authedUserContextString)
+	logrus.Debugf("Val: %v\n", val)
 	if valAsAuthedUser, ok := val.(*DelphisAuthedUser); ok {
+		logrus.Debugf("Does this ever return the User or just ID: %v   %+v\n", valAsAuthedUser.UserID, valAsAuthedUser.User)
```

Comment:
??

Reply:
what's this one? Sorry I'm not following!